frontend/app/.server/domain/dtos/benefit-renewal.dto.ts

```diff
@@ -6,11 +6,11 @@ export type BenefitRenewalDto = BenefitApplicationDto &
   ReadonlyDeep<{
     changeIndicators: {
       hasAddressChanged: boolean;
-      hasEmailChanged: boolean;
-      hasMaritalStatusChanged: boolean;
-      hasPhoneChanged: boolean;
-      hasFederalBenefitsChanged: boolean;
-      hasProvincialTerritorialBenefitsChanged: boolean;
+      hasEmailChanged?: boolean;
+      hasMaritalStatusChanged?: boolean;
+      hasPhoneChanged?: boolean;
+      hasFederalBenefitsChanged?: boolean;
+      hasProvincialTerritorialBenefitsChanged?: boolean;
```

Comment:
Is those optional props MUST be answered because calling the service method? If it's the case then they shouldn't be set as optional.

Reply:
These fields are required by the Adult/Child flow though.

Reply:
In this case, we should have a service methods and DTOs for each flows. What do you think?

Reply:
[Decoupled DTOs here](https://github.com/DTS-STN/canadian-dental-care-plan/compare/cb286c8b494e7a7d386ac597a9a57f1a8a08215a..df3e85eebaf994a3ad1f57438fe6d7cc73f2166e).

Service methods will be introduced in a future PR.